scripts/gas-refund-program/persistance/db-persistance.ts

```diff
@@ -53,7 +53,7 @@ export async function fetchTotalRefundedPSP(
       ...(toEpoch ? { epoch: { [Op.lt]: toEpoch } } : {}),
     },
     dataType: 'string',
```

Comment:
`dataType` overrides the resulting type?
And of course it's not type properly in the lib?
Of course

Reply:
problem with types yes